data/profiles/boot-livecd.ipxe

```diff
@@ -0,0 +1,4 @@
+kernel <%=repo%>/vmlinuz0 
+initrd <%=repo%>/initrd.img
+imgargs vmlinuz0 rootflags=loop initrd=initrd.img root=live:/<%=version%>.iso rootfstype=iso9660 !text !lang !ksdevice
```

Comment:
setup_iso.py automates the setup of the ISO on disk.  The installation looks like: /LiveCD/isoname, Mirroring the other OS flows, 'version' is 'isoname' and repo is '/LiveCD/isoname'.